The-SMIRNOFF-force-field-format.md

```diff
@@ -233,19 +233,19 @@ Here is an example:
 ```XML
 <ChargeIncrementModel version="0.3" number_of_conformers="10" quantum_chemical_method="AM1" partial_charge_method="CM2">
   <!-- A fractional charge can be moved along a single bond -->
-  <ChargeIncrement smirks="[#6X4:1]-[#6X3a:2]" chargeincrement1="-0.0073*elementary_charge" chargeincrement2="+0.0073*elementary_charge"/>
-  <ChargeIncrement smirks="[#6X4:1]-[#6X3a:2]-[#7]" chargeincrement1="+0.0943*elementary_charge" chargeincrement2="-0.0943*elementary_charge"/>
-  <ChargeIncrement smirks="[#6X4:1]-[#8:2]" chargeincrement1="-0.0718*elementary_charge" chargeincrement2="+0.0718*elementary_charge"/>
-  <!--- Alternatively, factional charges can be redistributed among any number of bonded atoms -->
-  <ChargeIncrement smirks="[N:1](H:2)(H:3)" chargeincrement1="+0.02*elementary_charge" chargeincrement2="-0.01*elementary_charge" chargeincrement3="-0.01*elementary_charge"/>
+  <ChargeIncrement smirks="[#6X4:1]-[#6X3a:2]" chargeincrement1="-0.0073*elementary_charge" chargeincrement2="0.0073*elementary_charge"/>
+  <ChargeIncrement smirks="[#6X4:1]-[#6X3a:2]-[#7]" chargeincrement1=+0.0943*elementary_charge" chargeincrement2="-0.0943*elementary_charge"/>
+  <ChargeIncrement smirks="[#6X4:1]-[#8:2]" chargeincrement1="-0.0718*elementary_charge" chargeincrement2="0.0718*elementary_charge"/>
+  <!--- Alternatively, fractional charges can be redistributed among any number of bonded atoms -->
+  <ChargeIncrement smirks="[N:1](H:2)(H:3)" chargeincrement1="0.02*elementary_charge" chargeincrement2="-0.01*elementary_charge" chargeincrement3="-0.01*elementary_charge"/>
 </ChargeIncrementModel>
 ```
 The sum of formal charges for the molecule or fragment will be used to determine the total charge the molecule or fragment will possess.
 
 `<ChargeIncrementModel>` provides several optional attributes to control its behavior:
 * The `number_of_conformers` attribute (default: `"10"`) is used to specify how many conformers will be generated for the molecule (or capped fragment) prior to charging.
 * The `quantum_chemical_method` attribute (default: `"AM1"`) is used to specify the quantum chemical method applied to the molecule or capped fragment.
-* The `partial_charge_method` attribute (default: `"CM2"`) is used to specify how uncorrected partial charges are to be generated from the quantum chemical wavefunction. Later additions will add restrained electrostatic potential fitting (RESP) capabilities.   
+* The `partial_charge_method` attribute (default: `"Mulliken"`) is used to specify how uncorrected partial charges are to be generated from the quantum chemical wavefunction. Later additions will add restrained electrostatic potential fitting (RESP) capabilities.   
```

Comment:
In our last discussion, I think we wanted to combine `quantum_chemical_method` and `partial_charge_method` to instead have a general two-stage procedure:
* Stage 1: Produce initial charges via some QM or ML method: AM1-Mulliken, graph convolutional network, VCharge, etc.
* Stage 2: Correct the initial charges with BCCs

That will require more significant revision of the proposed ChargeIncrementModel spec.

Reply:
Ok, I thought we had been trending toward this decision, but I couldn't find it in writing anywhere. I'm going to have `partial_charge_method` be the winner here (so, for example, `partial_charge_method="AM1-Mulliken"`). FOr the ToolkitRegistry, each toolkit will have a `supports_charge_method(method)` function which takes a `method` string and returns a bool indicating whether it can calculate charges using that method.

Reply:
The partial charge method may itself have other parameters, so it might be useful to split this into two separate tags for the different stages.

When we had discussed this, I think we discussed generalizing it into perhaps two tags: an initial charging tag (that might take optional parameters) and a BondChargeIncrement tag (that describes how to modify charges). 

Happy to chat further if you want to discuss ideas for making this flexible and simple to implement.